app/v3/notifications/rest.py

```diff
@@ -96,5 +111,13 @@ def v3_send_notification(request_data: dict) -> str:
     # This has the side effect of modifying the input in the upstream code.
     request_data["id"] = str(uuid4())
 
-    # TODO 1361 - Pass the validated request data to Celery by calling apply_async (or something else)
+    # Initiate a Celery task to process the validated request data.  This does not block.
+    # TODO - v2 uses the imported value "api_user" for the api_key.
```

Comment:
Are all of the TODOs spread throughout this code for the part 2 PR, or are they associated with follow-up tickets?

Reply:
Both, and I haven't written the follow-up tickets.  This ticket (#1361) has a list of tickets for additional work we know we need.  Part 2 of this ticket will handle error and retries for the Celery tasks.

Reply:
@EvanParish given that this is a WIP with a plan, I'm OK with the TODOs for now. On the final PR, I expect there to be no TODOs or -- if there are TODOs -- they contain ticket numbers.